### PR TITLE
File owner and permissions

### PR DESCRIPTION
**Pull request by areed**
### Description
Add annotations to control owner and permissions of certificates and key.

Fixes https://github.com/smallstep/autocert/issues/53.

💔Thank you!

### CODE REVIEW

**Review by maraino — CHANGES_REQUESTED** [review]
It looks good, but we need to add to the readme something about things to take care of, e.g. the renewer might fail if the mode is 0400 because it doesn't have write permissions, or because the user/group `step` does not have permissions to overwrite the file because it belongs to `postgres`

File: README.md [hunk 1]
```diff
@@ -171,6 +175,8 @@ spec:
       annotations:
         autocert.step.sm/name: hello-mtls-1h.default.svc.cluster.local
         autocert.step.sm/duration: 1h
+        autocert.step.sm/owner: "999:999"
```
Comment (maraino) [inline README.md:178]: Change this example to user:group instead of uid:gid

Comment (areed) [reply README.md:178]: I will update the docs to explain uid/gid is required.

**Comment by areed** [comment]
> It looks good, but we need to add to the readme something about things to take care of, e.g. the renewer might fail if the mode is 0400 because it doesn't have write permissions, or because the user/group `step` does not have permissions to overwrite the file because it belongs to `postgres`

Renewal worked fine for me in a test with `0400` and user `999:999`. The renewer container runs as root.

**Pull request merged**